tests/ops/util/test_tc_string.py

```diff
@@ -387,7 +387,7 @@ def test_build_tc_string_captify_accept_all(self, db):
         )
 
         assert model.cmp_id == 12
-        assert model.vendor_list_version == 19
+        assert model.vendor_list_version == 20
```

Comment:
When we move to live updates I'll refactor this test so we don't have to bump it every time.  It was handy to have some sort of test on this field to make sure it was being encoded/decoded properly, but it will need to be more dynamic.